packages/nc-gui/components/general/ReleaseInfo.vue

```diff
@@ -7,6 +7,9 @@ const { currentVersion, latestRelease, hiddenRelease } = useGlobal()
 
 const releaseAlert = computed({
   get() {
+    if (currentVersion.value?.includes('-beta.') || latestRelease.value?.includes('-beta.')) {
+      return false
+    }
```

Comment:
What if the user is having an old version, assume it's `0.92.0` and there are multiple releases like `0.93.0` and `0.94.0-beta.1`. Since the latest one is beta it won't show any notification I guess.